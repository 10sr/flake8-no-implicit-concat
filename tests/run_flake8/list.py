a = ["aaa",
     "bbb"
     "ccc"]


def f():
    a = ["aaa",
         "bbb"
         "ccc"]
    return a
