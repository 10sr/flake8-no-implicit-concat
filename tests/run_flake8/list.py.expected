./list.py:2:11: NIC001 Implicitly concatenated string literals
./list.py:8:15: NIC001 Implicitly concatenated string literals
